scripts/common_test.py

```diff
@@ -514,14 +555,14 @@ def _redirect_stdout(new_target):
         self.assertEqual(
             target_stdout.getvalue(), 'These\n\nare\n\nsample\n\nstrings.\n\n')
 
-    def test_install_npm_library(self):
+    def test_install_npm_library(self) -> None:
 
-        def _mock_subprocess_check_call(unused_command):
+        def _mock_subprocess_check_call(unused_command: str) -> None:
             """Mocks subprocess.check_call() to create a temporary file instead
             of the actual npm library.
             """
             temp_file = tempfile.NamedTemporaryFile()
-            temp_file.name = 'temp_file'
+            setattr(temp_file, 'name', 'temp_file')
```

Comment:
Why is this change needed?

Reply:
ditto elsewhere

Reply:
Probably the reason is that here we can't write an object attribute and I guess that's why I was getting this error
### error: Property "name" defined in "IO" is read-only  [misc]
And to resolve this I used `setattr` to set the attribute value.